src/SearchField/SearchFieldInput.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+
+import Input from '../Input';
+import { SearchFieldContext } from './SearchFieldAdvanced';
+
+const SearchFieldInput = (props) => {
+  const { className, ...others } = props;
+
+  const {
+    inputId, value, handleChange, handleFocus, handleBlur, refs,
+  } = useContext(SearchFieldContext);
+
+  return (
+    <Input
+      {...others}
+      ref={refs.input}
+      type="text"
+      role="searchbox"
+      id={inputId.current}
+      name="searchfield-input"
+      className={className}
+      value={value}
+      autoComplete="off"
```

Comment:
Should this be a default prop value?

Reply:
I will remove this as a default prop. 

If consumers of the component want to turn off autocomplete (which seems pretty common, afaik), they'd have to use the `SearchField.Advanced` implementation to be able to pass this `autoComplete` prop to the input.